src/chartmuseum/templates/_helpers.tpl

```diff
@@ -45,6 +45,9 @@ helm.sh/chart: {{ include "chartmuseum.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+{{- if .Values.commonLabels}}
```

Comment:
looking at the chart resources, it looks like the PV and PVC resources don't use these common labels so they won't pick these up. 

If you want the PV and PVC to be labeled, we'll need to update those with:

```
  labels:
    {{- include "chartmuseum.labels" . | nindent 4 }}
```